cssom-view/cssom-getClientRects-002.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>CSSOM View - GetClientRects().length is the same regardless source new lines</title>
+  <link rel="help" href="https://drafts.csswg.org/cssom-view-1/#dom-element-getclientrects">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+ </head>
+ <body>
+  <span id="single">
+   test test
+  </span><br/>
+  <span id="multiple">
+   test
+   test
+  </span>
+  <script>
+    test(function () {
+      const count = document.querySelector("#single").getClientRects().length;
+      const count2 = document.querySelector("#multiple").getClientRects().length;
+      assert_equals(count, 1, "count");
```

Comment:
Did you intentionally remove the whole description of the test?
I mean, `<title>` still has it, but I thought it would be beneficial to understand what exactly it was trying to test when you look at the results...